pkg/clients/eks/fargateprofile.go

```diff
@@ -23,7 +23,7 @@ import (
 	"github.com/google/go-cmp/cmp/cmpopts"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
-	"github.com/crossplane/provider-aws/apis/eks/v1alpha1"
+	v1alpha1 "github.com/crossplane/provider-aws/apis/eks/manualv1alpha1"
```

Comment:
can we use here `manualv1alpha1` without `v1alpha1` and change the code like in identityproviderconfig

`"github.com/crossplane/provider-aws/apis/eks/manualv1alpha1"`

Reply:
Fixed